config/config.js

```diff
@@ -88,6 +88,28 @@ var validateSecureMode = function (config) {
 };
 
 /**
+ * Validate Session Secret parameter is not set to default in production
+ */
+var validateSessionSecret = function (config, testing) {
```

Comment:
@lirantal agreed, but let's add this to the list of things to fix up in the formatting issue you are going to create.